kolibri/core/assets/src/utils/appCapabilities.js

```diff
@@ -18,6 +18,23 @@ export const checkCapability = key => store.getters.isAppContext && appCapabilit
 // checks and exposing the functions.
 
 export default {
+  /**
+   * @returns fn -> Promise<{ value: (boolean | null) }>
+   * Returns a function that returns a Promise that resolves to something responding to
+   * `data.value` whether it succeeds or fails.
+   */
+  checkIsMetered() {
+    if (!checkCapability('check_is_metered')) {
+      return Promise.resolve({ data: { value: null } });
```

Comment:
With the change below this could just be:
```suggestion
      return Promise.resolve(null);
```